src/ur3e_hande_robot_description/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ur3e_hande_robot_description</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```

Comment:
Add license and package description akin with readme.

Reply:
Extra info is added:
https://github.com/maffettone/erobs/blob/ce477aa837183f23017187b9c10a8649cb8814a1/src/ur3e_hande_robot_description/package.xml#L6-L8